src/ftdc_tools/rollups/client_perf.py

```diff
@@ -58,13 +78,21 @@ def add_doc(self, doc: FTDCDoc) -> None:
         else:
             duration = float(doc["timers"]["duration"])
         extracted_duration = duration - self.previous_duration
-
+        number_of_ops = doc["counters"]["ops"] - self.previous_ops
+        number_of_ops = (
+            1 if number_of_ops == 0 else number_of_ops
```

Comment:
Why are we making number of ops 1 in this case? Shouldn't we keep 0 ops if 0 were reported?

Reply:
To handle the same case mentioned in the previous comment.

Reply:
fixing this as well